docs/userstories/README.md

```diff
@@ -1,94 +0,0 @@
-# Network Policy API User Stories
-
-This directory should be the entry point for any new object or feature proposed 
-to the sig network-policy-api subgroup 
-
-It is meant to house all user stories that have been reviewed and agreed upon by
-the sig for any new k8's networking security object. The model should follow a 
-standard github workflow, with a directory representing proposed new objects 
-which contains individual user stories that can be reviewed and commented on. 
-
-All work proposed and merged here should follow the [style guidelines](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md) 
-agreed upon by the Kubernetes community
-
-## Directory Structure 
-
-Each new Object should have it's own directory, following directory naming 
-conventions seen in [`kubernetes/kubernetes`.](https://github.com/kubernetes/kubernetes) Specifically the directory name 
-should be **lowercase** and **descriptive**. The directory structure should 
-roughly represent whether or not the new object will augment the existing [network policy API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#networkpolicy-v1-networking-k8s-io) (`network-policy-api/docs/userstories/v1/<newobject>/`) or create a 
-new set of APIs (`network-policy-api/docs/userstories/v2/<newobject>/`). 
-If there is some doubt as to which bucket the new object will fall into, the new 
-directory should be made outside of either subdirectory (`network-policy-api/docs/userstories/<newobject>/`) and a final 
-determination can be made at a later date.
-
-## User Story Structure
-
-To submit a new user story please 
-open a PR roughly following the [template](#template) shown below. Each story 
-should be in it's own markdown file titled with a descriptive 3 word **lowercase**
-title:  `<word1>_<word2>_<word3>.md, ...`
-
-The following template should be used as a suggested guideline, please feel free
-to deviate and augment as needed. 
-
-### Template
-
-```markdown
-
-### Summary 
-
-A user story should typically have a summary structured this way:
```

Comment:
@npinaeva Is this blurb useful at all for the NPEP template? If not no worries just want to check in before deleting

Reply:
I think we could copy this part to the NPEP template, but not a strong opinion